blaze/compute/pyfunc.py

```diff
@@ -63,7 +64,7 @@ def _print_python(expr, leaves=None):
 
 @dispatch((datetime.datetime, datetime.date))
 def _print_python(expr, leaves=None):
-    return repr(expr), {'datetime': datetime}
+    return repr(expr), {'datetime': datetime, 'Timestamp': pd.Timestamp}
```

Comment:
> Are there cases where we intended to print python datetime objects that we actually end up printing Pandas Timestamps?

Not sure. FWIW, `Timestamp` is a subclass of `datetime.datetime`.

> Do we ever not assume that pandas is around?

No, pandas is a hard dependency.